packages/server/graphql/mutations/endRetrospective.ts

```diff
@@ -80,10 +93,20 @@ const finishRetroMeeting = async (meeting: MeetingRetrospective, context: GQLCon
             .default(0) as unknown as number,
           topicCount: reflectionGroupIds.length,
           reflectionCount: reflections.length,
+          summary,
```

Comment:
I think this should fix: https://github.com/ParabolInc/parabol/issues/7674

Previously, we were updating the Rethinkdb meeting object both here and in `generateWholeMeetingSummary`. By doing it in one place, we'll hopefully fix that race condition

Reply:
Bug fix moved to this PR: https://github.com/ParabolInc/parabol/pull/8197